sbin/build.sh

```diff
@@ -235,11 +235,16 @@ removingUnnecessaryFiles()
 
 createOpenJDKTarArchive()
 {
-  GZIP=-9 tar -czf OpenJDK.tar.gz ./j2sdk-image
-
-  mv OpenJDK.tar.gz $TARGET_DIR
-
-  echo "${good}Your final tar.gz is here at ${PWD}${normal}"
+  case "${OS_MACHINE_NAME}" in
```

Comment:
OS_MACHINE_NAME stores / returns strings like 'x86_64' - im not sure how this works - is the windows machine returning 'cygwin' instead of 'x86_64' in OS_MACHINE_NAME ?